[DOCS] More doc updates to remove the CLI (#7874)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/create_expectations_overview.md

### Creating Expectations interactively

> We provide an in-depth guide for interactively creating Expectations in our guide on [how to create and edit Expectations with instant feedback from a sample batch of data](./how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md).

Reviewer: ```suggestion
To interactively create Expectations, see [How to create and edit Expectations with instant feedback from a sample batch of data](./how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md).
```
